src/Handles/Handles.js

```diff
@@ -3,12 +3,19 @@ import PropTypes from 'prop-types'
 import { callAll } from '../utils'
 
 class Handles extends Component {
+  autofocus = e => {
+    e.target.focus()
+  }
+
   getHandleProps = (id, props = {}) => {
-    const { emitMouse, emitTouch } = this.props
+    const { emitKeyboard, emitMouse, emitTouch } = this.props
 
     return {
       ...props,
-      onMouseDown: callAll(props.onMouseDown, e => emitMouse(e, id)),
+      onKeyDown: callAll(props.onKeyDown, e => emitKeyboard(e, id)),
+      onMouseDown: callAll(props.onMouseDown, this.autofocus, e =>
```

Comment:
I added this small `autofocus` method to make it behave like the native `<input type="range"/>`: clicking on it will focus it too.

Reply:
Hey, this looks good.  Need a few days before I can go over this and try it out.  Is it safe to assume that there's a focus method on the target?  I think so, but not a 100% on that.  With that autofocus is there still an advantage using a button vs div? Just curious.  So you can't control whether it will autofocus on the handle on mousedown with these changes, right?  Trying to think if that's going to be an issue for any use case.  Could allow disabling that with a prop.

Reply:
A `div` won't get the focus despite using using `.focus()` unless it has a `tabindex` but eh method won't throw an error. All `HTMLElement` have `.focus()` event IE 9 so we should be fine, but I can at least test if its a function before calling it.

Since it's an usability improvement and it makes this component behave like the native one, I would rather NOT make it configurable. But maybe there is a valid reason to prevent autofocus that I can't think of, I'll ask on twitter, hopefully I have accessibility experts following me :p

Reply:
Ok.  Thanks for getting back to me on those points.  I'm going to merge this as is. Looks good to me.